components/rss/actions/merge-rss-feeds/merge-rss-feeds.js

```diff
@@ -0,0 +1,94 @@
+const Parser = require('rss-parser');
+
+module.exports = {
+  name: "Merge RSS Feeds",
+  description: "Retrieve multiple RSS feeds and return a merged array of items sorted by date.",
+  key: "rss-merge-rss-feeds",
+  version: "0.2.1",
+  type: "action",
+  props: {
+	feeds: {
+		type:"string[]",
+		label:"Feeds",
+		description:"The list of RSS feeds you want to parse."
+	},
+	merge: {
+		type:"boolean",
+		optional:true,
+		default:true,
+		description:"If true, all items are returned in a date sorted array. If false, each feed is returned as one result in the array."
+	},
+	rss: {
+		type: "app",
+		app: "rss"
+	}
+  },
+  async run() {
+
+	const getFeeds = async function(url) {
+		return new Promise((resolve, reject) => {
+			let parser = new Parser();
+			resolve(parser.parseURL(url));
```

Comment:
`parse.parseURL` should return a Promise, so I believe you can remove this function entirely. Below, where you `push` to the `requests` array, you should be able to just use:

```
requests.push(parser.parseURL(this.feeds[i]))
```

(I haven't tested, just looking at docs)